patches/minecraft/net/minecraft/client/gui/screen/inventory/CreativeScreen.java.patch

```diff
@@ -69,23 +68,22 @@
 +         field_147002_h.func_148329_a(0.0F);
 +         return;
 +      }
-+
        String s = this.field_147062_A.func_146179_b();
        if (s.isEmpty()) {
           for(Item item : Registry.field_212630_s) {
-@@ -375,9 +409,9 @@
+@@ -375,9 +407,9 @@
  
     protected void func_230451_b_(MatrixStack p_230451_1_, int p_230451_2_, int p_230451_3_) {
        ItemGroup itemgroup = ItemGroup.field_78032_a[field_147058_w];
 -      if (itemgroup.func_78019_g()) {
-+      if (itemgroup != null && itemgroup.func_78019_g()) {
++      if (itemgroup.func_78019_g() && itemgroup != null) {
```

Comment:
Revert this, it is breaking the entire use case of this not null check in the first place

Reply:
oops, didn't realize I had changed the order.